frameworks/kafka/docs/security.md

```diff
@@ -243,3 +197,6 @@ Install the DC/OS Apache Kafka service with the following options in addition to
 ```
 
 `service.security.authorization.super_users` should be set to a semi-colon delimited list of principals to treat as super users (all permissions). The format of the list is `User:<user1>;User:<user2>;...`. Using Kerberos authentication, the "user" value is the Kerberos primary, and for SSL authentication the "user" value is the `CN` of the certificate. The Kafka brokers themselves are automatically designated as super users.
+
+<!-- TODO. @Evan, did you write something for this already? Or am I mis-remembering? -->
```

Comment:
No, I did not: https://jira.mesosphere.com/browse/INFINITY-3104

Reply:
(although this could be sufficient)

Reply:
I will leave it up to you to decide.